src/index-node.ts

```diff
@@ -1,14 +1,56 @@
 import 'cross-fetch/polyfill'
 import { FiatConnectClientImpl } from './fiat-connect-client'
-import { FiatConnectClientConfig } from './types'
-import fetchCookie from 'fetch-cookie'
-export * from './types'
+import { AddKycParams, FiatConnectClientConfig, ResponseError } from './types'
+import makeFetchCookie from 'fetch-cookie'
+import { CookieJar } from 'tough-cookie'
+import { KycSchema, KycStatusResponse } from '@fiatconnect/fiatconnect-types'
+import { Result } from '@badrap/result'
+import { handleError } from './fiat-connect-client'
 
 export class FiatConnectClient extends FiatConnectClientImpl {
   constructor(
     config: FiatConnectClientConfig,
     signingFunction: (message: string) => Promise<string>,
+    cookieJar?: CookieJar.Serialized,
   ) {
-    super(config, signingFunction, fetchCookie(fetch))
+    if (cookieJar) {
+      super(
+        config,
+        signingFunction,
+        makeFetchCookie(fetch, CookieJar.deserializeSync(cookieJar)),
+        cookieJar,
+      )
+    } else {
+      super(config, signingFunction, makeFetchCookie(fetch))
+    }
+  }
+
+  /**
+   * https://github.com/fiatconnect/specification/blob/main/fiatconnect-api.md#3421-post-kyckycschema
+   * Same as implementation in fiat-connect-client.ts except check for _ensureLogin
+   */
+  async addKyc<T extends KycSchema>(
```

Comment:
I am not sure if this is  the right place for this method. Do we want the cookie support only on the node implementation? I think this should just be on FiatConnectClientImpl. Also, this implementation should be generic to all methods that require login, not just addKyc. cc @jophish